ios/RNInCallManager/RNInCallManager.m

```diff
@@ -236,7 +236,48 @@ - (void)dealloc
 
 RCT_EXPORT_METHOD(setSpeakerphoneOn:(BOOL)enable)
 {
-    NSLog(@"RNInCallManager.setSpeakerphoneOn(): ios doesn't support setSpeakerphoneOn()");
+     BOOL success;
+     NSError *error = nil;
+	 NSArray* routes = [_audioSession availableInputs];
+
+    if(!enable){
+	    NSLog(@"Is on Earpiece ");
```

Comment:
The log message is a bit off. Do you mean "Checking if earpiece is on."? If yes update the message to be just that.

Reply:
added suitable log messages and done some code formatting changes